### ISSUE TITLE
Visual for buy targets (something similar to what zeno gui has or at least better feedback, full buy log and targets)

### ISSUE THREAD

**Issue by Utone** [taniman/profit-trailer#824]
So far it's been fishing in the dark, having trouble finding fitting strategies for current market conditions and empty buy logs until, you tweak, tweak aaaand tweak some more until you fall asleep again..lol

Having a visual of where you are aiming would be nice.

**Comment by nstefi** [comment]
Yes, this is the feature I dreamed about for a few weeks now.
I heard that Gunbot has some plugin to Tradingview and I saw a video where you could see the dots on the chart where the buys and sells took place.
I thought it would be nice to have some small charts, each pair would need one, and you could adjust the trailing parameters and show some historical chart with some simulated buys and sells and show the dots on the chart. It would be nice to have that as a control panel page where you adjust the sliders for each parameter and show some estimated results for let's say last 7 days.
Also a lot of parameters are dependent on each other. If you modify one, you need to be careful of the rest too.
Or at least show some kind of statistics numbers, let's say when you have a dip, followed by a local high, what's the difference (the height of that local peak), so you could adjust your ALL_trailing_buy parameter to not be higher than the average of that (let's say over last 7 days).

**Comment by cristianserban** [comment]
guys please come with concrete requests. maybe some example ..

**Comment by nstefi** [comment]
Hey, it's your product. You should do some Google search too. I thought I already gave you a few clues.
Here are some examples for you:
Look at the small charts from coinmarketcap.com, you could show some small charts like that for each pair. And when the user clicks on the chart, you could show them some small arrows like in this video:
[https://www.youtube.com/watch?v=Tp3hnTRoFd8](url)
It doesn't have to be a huge chart like Tradingview's, a small one is enough as long as you see the shape of the chart and you could see the places where the bot would buy and sell.
I thought it would be best to put these in a control panel page where you set all your parameters, and show the effect of those based on a chart for last 7 days. Or at least if you don't show charts, you could show some statistics or counts of buys and sells that the bot would have done in last 7 days based on your new parameters. I have read articles of other bots and I think they call that feature backtesting.
So basically the bot should help you setting the parameters.
Thanks.

**Comment by nstefi** [comment]
I added comments, why is it still closed?

**Comment by taniman** [comment]
Well i will not reopen this. This is not something we want to invest time in currently. The bot needs to be a stable bot. You can check charts on yuor exchange and tweak based on that. untill we think it's time to implement such a feature @nstefi

**Comment by nstefi** [comment]
OK, thanks. But currently it's really hard to configure. And it's very sensitive to settings. If you make small changes, it can have big impact. There is a high chance that pairs go into DCA. When I look at the Sales Log, I see more pairs sold out of DCA then without DCA, so that means it doesn't really buy at the right time, because most times the price is falling under. So to me it's more like a gambling and it relies on DCA. The buys need to be improved.
